src/rapids_singlecell/preprocessing/_utils.py

```diff
@@ -72,6 +75,127 @@ def _mean_var_minor(X, major, minor):
     return mean, var
 
 
+@with_cupy_rmm
+def _mean_var_minor_dask(X, major, minor):
+    """
+    Implements sum operation for dask array when the backend is cupy sparse csr matrix
+    """
+
+    from rapids_singlecell.preprocessing._kernels._mean_var_kernel import (
+        _get_mean_var_minor,
+    )
+
+    get_mean_var_minor = _get_mean_var_minor(X.dtype)
+    get_mean_var_minor.compile()
+
+    def __mean_var(X_part):
+        mean = cp.zeros(minor, dtype=cp.float64)
+        var = cp.zeros(minor, dtype=cp.float64)
+        block = (32,)
+        grid = (int(math.ceil(X_part.nnz / block[0])),)
+        get_mean_var_minor(
+            grid, block, (X_part.indices, X_part.data, mean, var, major, X_part.nnz)
+        )
+        return cp.vstack([mean, var])[None, ...]  # new axis for summing
+
+    n_blocks = len(X.to_delayed().ravel())
+    mean, var = X.map_blocks(
+        __mean_var,
+        new_axis=(1,),
+        chunks=((1,) * n_blocks, (2,), (minor,)),
+        dtype=cp.float64,
+        meta=cp.array([]),
+    ).sum(axis=0)
+    var = (var - mean**2) * (major / (major - 1))
+    return mean, var
+
+
+# todo: Implement this dynamically for csc matrix as well
```

Comment:
seems like an easy todo, just `.T`, use the function, `.T` back, right?

Reply:
not that easy for dask

Reply:
I'll try to adress this in a later PR